src/components/PopupWithForm.js

```diff
@@ -33,4 +33,8 @@ export default class PopupWithForm extends Popup {
     });
     super.setEventListeners();
   }
+  close() {
+    super.close();
+    this._popupForm.reset();
```

Comment:
You should **clear ** the inputs only after **submitting** a card rather than when you open or close popups because if you unexpectedly close the popup, you will be upset if the inputs are empty when you open it again, and you’ll have to type the data again.